### PR TITLE
LibAudio: Set asynchronous audio enqueuer thread to maximum priority

### PR DESCRIPTION
**Pull request by kleinesfilmroellchen**
Chunk 5 of #16049

Anything that handles audio in this way should run at maximum priority.

**Note regarding Ports**: This may break ports that use audio. @GMTA informed me that at least quake3 uses pledge and I could not find any other ports which use pledge + audio with a quick search. I can not get quake3 to install, therefore I don't know whether this is a problem.

### CODE REVIEW

File: Userland/Utilities/aplay.cpp [hunk 1]
```diff
@@ -64,10 +64,6 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 
     // If we're downsampling, we need to appropriately load more samples at once.
     size_t const load_size = static_cast<size_t>(LOAD_CHUNK_SIZE * static_cast<double>(loader->sample_rate()) / static_cast<double>(audio_client->get_sample_rate()));
-    // We assume that the loader can load samples at at least 2x speed (testing confirms 9x-12x for FLAC, 14x for WAV).
-    // Therefore, when the server-side buffer can only play as long as the time it takes us to load a chunk,
-    // we give it new data.
-    unsigned const min_buffer_size = load_size / 2;
```
Comment (TobyAsE) [inline Userland/Utilities/aplay.cpp:70]: This seems unrelated to the thread priority.

File: Userland/Utilities/aplay.cpp [hunk 2]
```diff
@@ -115,7 +111,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
                 // We're done and the server is done
                 break;
             }
-            while (audio_client->remaining_samples() > min_buffer_size) {
+            while (audio_client->remaining_samples() > load_size) {
```
Comment (TobyAsE) [inline Userland/Utilities/aplay.cpp:114]: This as well.

File: Userland/Utilities/aplay.cpp (hunk 1 above)
Comment (kleinesfilmroellchen) [reply Userland/Utilities/aplay.cpp:70]: Yeah, that should have been its own commit.

**Review by ADKaster — APPROVED** [review]
It looks like at least the quake port already has the proc pledge, so we'll see if anything else breaks <https://github.com/SerenityPorts/SerenityQuake/commit/a7b8aea818121f1351d773ed9265ba1b41ba880f>
